soh/soh/GameMenuBar.cpp

```diff
@@ -1563,9 +1567,18 @@ namespace GameMenuBar {
             }
             ImGui::PopStyleVar(3);
             ImGui::PopStyleColor(1);
+#ifdef ENABLE_CROWD_CONTROL
             UIWidgets::PaddedEnhancementCheckbox("Crowd Control", "gCrowdControl", true, false);
             UIWidgets::Tooltip("Requires a full SoH restart to take effect!\n\nEnables CrowdControl. Will attempt to connect to the local Crowd Control server.");
 
+            // TODO: Better place for this check? Hooks?
```

Comment:
Probably hooks, or maybe if you expose the isEnabled you could do something like `if (CVar_Get() != CC::Instance->IsEnabled())` and only change it then. 

Currently this calls the enable or disable method every frame when this menu is open right?